### ISSUE TITLE
Add ability to catch libsml error messages

### ISSUE THREAD

**Issue by andig** [volkszaehler/libsml#17]
...instead of writing them to STDERR

**Comment by hmueller01** [comment]
You are right. A library should not write to strerr. It should return an error number and the application should decide what to do (write an error message, stop execution, nothing ...)
BTW: If you like to suppress the error messages right now you can use `sml_server /dev/ttyUSB0 2>/dev/null`

**Comment by r00t-** [comment]
realize that before https://github.com/volkszaehler/libsml/pull/12 it would even write them to stdout.

**Comment by r00t-** [comment]
made some implementation, please have a look at #93
